lib/main.coffee

```diff
@@ -351,10 +351,10 @@ module.exports = Hydrogen =
             @wsKernelPicker = new WSKernelPicker (kernel) =>
                 @clearResultBubbles()
 
-                grammar = @editor.getGrammar()
+                grammar = kernel.grammar
```

Comment:
Oops, I'm sorry I didn't read `ps aux` correctly. I had an other python process running and thought it was related to hydrogen :wink:.